registration-processor/registration-processor-landing-zone-Utility/src/main/java/io/mosip/registration/processor/util/LandingZoneUtility.java

```diff
@@ -0,0 +1,127 @@
+@Component
+class LandingZoneUtility{
+	private static Logger regProcLogger = RegProcessorLogger.getLogger(LandingZoneUtility.class);
+	private static final int MAX_NUMBER_OF_PACKETS = 100;
+	@Value("${landing.zone.account.name}")
+	private String landingZoneAccount;
+		
+	@Value("${landing.zone.type:ObjectStore}")
+	private String landingZoneType;
+	
+	@Value("${registration.processor.packet.ext}")
+    private String extention;
+	
+	@Autowired
+	private FileManager<DirectoryPathDto, InputStream> fileManager;
+	
+	@Autowired
+	private ObjectStoreAdapter objectStoreAdapter;
+	
+	
+	@Autowired
+	Environment env;
+	 /**
+	  *   
+	  * moves the packets from dmz server to objectstore if the landing zone has been changed to 
+      */
+	@Scheduled(fixedDelayString = "${mosip.regproc.landing.zone.fixed.delay.millisecs:43200000}",
+	            initialDelayString = "${mosip.regproc.landing.zone.inital.delay.millisecs:300000}")
+	public void movePacketsToObjectStore() {
+		
+		if(landingZoneType.equalsIgnoreCase(LandingZoneTypeConstant.OBJECT_STORE)) {
+			regProcLogger.debug(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.USERID.toString(),
+					"", "PacketUploaderServiceImpl::movePacketsToObjectStore()::entry");
+			
+			 try  {
+				
+				 List<File> packetList=new ArrayList<>(); 
+				 Files.list(Paths.get(env.getProperty(DirectoryPathDto.LANDING_ZONE.toString())))
+						    .map(Path::toFile)
+						    .filter(File::isFile)
+						    .filter(file-> file.getName().endsWith(extention))
+						    .forEach(file -> {
+						    	packetList.add(file);
+						 		if(packetList.size()>=MAX_NUMBER_OF_PACKETS) {
+						 			packetList.forEach(packet->handlePacket(packet));
+						 			packetList.clear();
+						 		}
+						    });
+				 
+				 if(packetList.size()<MAX_NUMBER_OF_PACKETS) {
+					 packetList.forEach(packet->handlePacket(packet));
+			 			packetList.clear();
+				 }
+				
+				}catch (Exception e) {    
+					     regProcLogger.error(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.REGISTRATIONID.toString(),
+					        		   "", e.getMessage());
+					 }
+				}
+				
+			
+			regProcLogger.debug(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.USERID.toString(),
+					"", "PacketUploaderServiceImpl::movePacketsToObjectStore()::exit");
+	}
+		
+	private void handlePacket(File packet) {
+		   
+		   String regId=packet.getName().split("-")[0];
+		   String packetId=packet.getName().split("\\.")[0];
+		   if(packet.exists()) {
+			   boolean result;
+			try {
+				InputStream stream=new FileInputStream(packet);
+				result = objectStoreAdapter.putObject(landingZoneAccount,regId, null, null, packetId, stream);
+				stream.close();
+			   if(!result) {
+					regProcLogger.error(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.USERID.toString(),
+									 regId, packetId+":Packet store not accesible");
+				}
+				else {
+					fileManager.deletePacket(DirectoryPathDto.LANDING_ZONE, packetId);
```

Comment:
Did you verify if delete is working? i remember we had simmilar functionality before which didnot work but not sure if we were using same fileManager.

Reply:
It is working in local.